google/cloud/storage/tests/testbench.py

```diff
@@ -896,6 +1005,23 @@ def lookup_bucket(bucket_name):
     return bucket
 
 
+def lookup_object(bucket_name, object_name):
+    """
```

Comment:
Have you run the linter on this?  (I thought it would throw a formatting fit for this description not being on the same line as the first triple-quote)

Reply:
I have been running `yapf` to reformat the code. If you think we should use a different tool I am all for it, cannot say that I care one way or the other.